flimfitgen.py

```diff
@@ -34,8 +35,8 @@ def usage():
     "/MCR_R2015b_maci64_installer.zip"
 
 for x, y in (
-        ("FLIMFIT_51_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
-        ("FLIMFIT_51_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
+        ("FLIMFIT_52_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
+        ("FLIMFIT_52_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
```

Comment:
I have no strong feelings on the file names Whatever works for you but my guess would be that we're unlikely to return to releasing parallel versions for different OMERO's so the "_52" bit is probably superfluous.